src/shorts/shorts.route.js

```diff
@@ -1,10 +1,11 @@
 import express from 'express';
 import asyncHandler from 'express-async-handler';
-import { createShorts, getShortsDetail, searchShorts } from './shorts.controller.js';
+import { createShorts, getShortsDetail, searchShorts, addComment } from './shorts.controller.js';
 
 export const shortsRouter = express.Router({mergeParams:true});
 
 shortsRouter.get('/:shortsId', asyncHandler(getShortsDetail));
 shortsRouter.get('', asyncHandler(searchShorts));
 
-shortsRouter.post('', asyncHandler(createShorts));
\ No newline at end of file
+shortsRouter.post('', asyncHandler(createShorts));
+shortsRouter.post('/comments', asyncHandler(addComment));
```

Comment:
```suggestion
shortsRouter.post('/:shortsId/comments', asyncHandler(addComment));
```
명세에 맞춰 위와 같이 수정 부탁드립니다!